### DISCUSSION TITLE
Organization Migration Plan

### DISCUSSION THREAD

**Discussion by AdrienPoupa** [VinylMusicPlayer/VinylMusicPlayer#641] [General]
Followup of https://github.com/VinylMusicPlayer/VinylMusicPlayer/pull/555#issuecomment-1403759265

To make Vinyl development more resilient:
- [x] Transfer the repository into an organization
- [x] Update FDroid metadata: https://gitlab.com/fdroid/fdroiddata/-/merge_requests/12491
- [x] Automate Play Store release: https://github.com/marketplace/actions/upload-android-release-to-play-store

**Comment by soncaokim** [comment]
The google play one is the main blocker. 

I havent checked but I believe they require api>=31, and we cannot upgrade beyond 29 (cf past discussion).

Nevertheless, it's worth to automate as much as possible this part.

**Comment by soncaokim** [comment]
Just for your info, I'll try in the next few days to merge back the changes from my fork to this repo.
And I'm counting roughly 10 incoming PRs -> your review is needed (especially on new features)

**Comment by soncaokim** [comment]
@AdrienPoupa I'd like to enable the master branch protection (i.e. only allow merge commit, etc). More restrictive but more protective. Is that OK with you?

**Reply by AdrienPoupa** [reply]
Sure, it is a great idea

**Reply by soncaokim** [reply]
Done

**Comment by Octoton** [comment]
For the api limitation, so time ago I manage a proof of concept to enable api 30+, part of playlist management is still an issue in it and some part of the code is not clean but can push it to help (stop working on it beacuse of time cosntraint and not code limitation).

**Reply by soncaokim** [reply]
Well, I've started a trial on an internal playlist mgmt (disconnected from Android Media Store). The proof of concept works but I was considering that the disconnection a major breaking change hence didnt pursue the effort.

**Reply by Octoton** [reply]
yes I saw this, what I did was enabling removal/editing of song and album (see https://github.com/VinylMusicPlayer/VinylMusicPlayer/pull/643). With both proof of concept, the user experience is quite nice
